readme.md

```diff
@@ -74,6 +76,15 @@ install location to `resources/ovmf.fd`.
     Where `<version>` and `<architecture>` are the any of the versions or
     architectures available in the above table.
 
+    To target a snapshot, override the `checksum` variable manually by
+    specifying `-var checksum=CHECKSUM` at the end when invoking the `build.sh`
```

Comment:
Please use `<checksum>` to indicate a placeholder that should be replaced when running the command. This is consistent with the rest of the documentation.

Reply:
https://github.com/cross-platform-actions/openbsd-builder/pull/19/commits/5d6a0d5857991fe26a0d2dd6537d3678e074cc96